_data/projects/Firecracker.yml

```
@@ -0,0 +1,10 @@
+name: Firecracker
+desc: CLI that maps business logic, anomaly decider, attacker and visualizer
+site: https://github.com/blst-security/firecracker
+tags:
+  - API-securrity
+  - Rust
+  - CLI
+upforgrabs:
+  name: up for grabs
```

Please update this with whatever label you wish to use:

```suggestion
  name: help wanted
```